lesson04/assignment/css/style.scss

```diff
@@ -0,0 +1,118 @@
+$font: sans-serif;
+$navColor: #343a40; // navigation and footer color
+$companyLinksPadding: 20px;
+
+body {
+  margin: 0;
+  font-family: $font;
+}
+
+// centered text settings 
+@mixin centerText {
+  color:#fff;
+  text-align: center;
+  font-family: $font;
+  font-weight: bold;
+}
+
+// Controls the color of the links on the page
+// Makes the "home" button color white so user's can see it's the page they're currently on
+// Makes the link white on hover
+@mixin linkColor($color: #fff) {
+  & a:hover {
+    color: $color;
+  }
+  & .active {
+    color: $color;
+  }
+}
+
+// Navigation bar
+nav {
+  @include linkColor;
+  background-color: $navColor;
+  
+  & ul {
+    display: flex;
+    margin: 0;
+    list-style: none;
+    padding: 20px;
+  }
+
+}
+
+nav li,
+nav a {
+  color: rgba(255,255,255,.5);
+  padding: 10px;
+  text-decoration: none;
+  font-weight: 900;
+}
+
+// Styles the background image at the top of the page
+header {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("../imgs/hero-image.webp");
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 2em 5em 7em 5em;
+  text-align: center;
+
+  & h1 {
+    padding: 60px;
+    font-size: 1.5rem;
+    font-style: italic;
+    @include centerText;
+    animation: slideIn 2s ease-out 0s 1;
```

Comment:
I like the use of the ease-out timing, you may want to experiment with shortening the animation duration.